pegasus/sites.v3/code.org/views/visit_partial_event_logger.haml

```diff
@@ -4,6 +4,7 @@
   def partial_url_to_event_map 
     {
       '/ai': AnalyticsConstants::AI_PAGE_VISIT_EVENT,
+      '/ai/pl/101': AnalyticsConstants::AI_PL_PAGE_VISIT_EVENT,
```

Comment:
I don't believe this line is necessary since you added the `= view` line in the `.haml` file. These lines are here to check the url's of `.md.partial` files since this was the only way I could get the Amplitude events to trigger on those types of files

Reply:
Ah, makes sense! Thank you